pricing/v1/pricing.proto

```diff
@@ -46,23 +46,34 @@ message GetInfoResponse {
 }
 
 // Request message for GetPricingInfo
-message GetPricingInfoRequest {
+message GetPricingRequest {
+
+  //Required cloud (AWS/Azure)
```

Comment:
Would be helpful if you include a guide as to what actual strings clients can use as value for this field (e.g. aws, azure). Using all lowercase would be more convenient. State that we only support aws and azure for now.